chat-plugins/othermetas.js

```diff
@@ -105,9 +105,10 @@ exports.commands = {
 		for (let i in pokeobj.baseStats) {
 			bst += pokeobj.baseStats[i];
 		}
+		if (bst > 350) return this.sendReply(`|html|${Chat.getDataPokemonHTML(pokeobj)}`);
 		let newStats = {};
 		for (let i in pokeobj.baseStats) {
-			newStats[i] = pokeobj.baseStats[i] * (bst <= 350 ? 2 : 1);
+			newStats[i] = pokeobj.baseStats[i] * 2;
```

Comment:
First of all [loop-invariant code motion optimization](https://en.wikipedia.org/wiki/Loop-invariant_code_motion) is a thing. Second, even if that optimization wasn't a thing, comparing to the cost of loop over a map (which happens just above), cost of comparing an integer (pretty much one CPU instruction, three considering conditional moves here) is irrelevant.